### PR TITLE
Feat : Add Label Form On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review add labels form code
Add-label

### CODE REVIEW

File: src/components/AddLabelForm/AddLabel.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+                        </div>
+
+                        {
+                            labelArr.length > 0 &&
+                            <div className="chips-container">
+                                {
+                                    labelArr.map(({ value }, i) =>
+                                        <button key={i} className="chips-btn" >
+                                            <p>{value}</p> <Icon className="iconify cursor_" onClick={(e) => deleteLableHandler(e, value, setLabelArr)} icon="akar-icons:circle-x" />
```
Comment (Mohasin-Haque) [inline src/components/AddLabelForm/AddLabel.jsx:66]: Is it a good practice to write class name like this?

**Review by Mohasin-Haque — APPROVED** [review]
Everything looks good to me. You can merge it.

File: src/components/AddLabelForm/AddLabel.jsx (hunk 1 above)
Comment (Rishi05051997) [reply src/components/AddLabelForm/AddLabel.jsx:66]: Ya no any issue with that

**Review by junaidshaikh-js — APPROVED** [review]
Good to merge.

**Pull request merged**